### PR TITLE
Add mean and min

### PR DESCRIPTION
**Pull request by denisovap2013**
Added support for:

- Min
- Max
- Mean
- Sum

And corresponding tests

### CODE REVIEW

File: onnx2torch/node_converters/sum.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+from onnx2torch.node_converters.registry import add_converter
+from onnx2torch.onnx_graph import OnnxGraph
+from onnx2torch.onnx_node import OnnxNode
+from onnx2torch.utils.common import OperationConverterResult
+from onnx2torch.utils.common import onnx_mapping_from_node
+from onnx2torch.utils.custom_export_to_onnx import CustomExportToOnnx
+
+
+class OnnxSum(nn.Module):  # pylint: disable=missing-docstring
```
[the remark this answers is not in the document]
Comment (denisovap2013) [reply onnx2torch/node_converters/sum.py]: Delegated to the class `OnnxBaseElementWise`

File: onnx2torch/node_converters/mean.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+from onnx2torch.node_converters.registry import add_converter
+from onnx2torch.onnx_graph import OnnxGraph
+from onnx2torch.onnx_node import OnnxNode
+from onnx2torch.utils.common import OperationConverterResult
+from onnx2torch.utils.common import onnx_mapping_from_node
+from onnx2torch.utils.custom_export_to_onnx import CustomExportToOnnx
+
+
+class OnnxMean(nn.Module):  # pylint: disable=missing-docstring
```
[the remark this answers is not in the document]
Comment (denisovap2013) [reply onnx2torch/node_converters/mean.py]: > OnnxToTorchModuleWithCustomExport

Delegated to the class `OnnxBaseElementWise`

File: onnx2torch/node_converters/min_max.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+from onnx2torch.node_converters.registry import add_converter
+from onnx2torch.onnx_graph import OnnxGraph
+from onnx2torch.onnx_node import OnnxNode
+from onnx2torch.utils.common import OperationConverterResult
+from onnx2torch.utils.common import onnx_mapping_from_node
+from onnx2torch.utils.custom_export_to_onnx import CustomExportToOnnx
+
+_OPERATORS = {
+    'Min': torch.min,
+    'Max': torch.max,
```
[the remark this answers is not in the document]
Comment (denisovap2013) [reply onnx2torch/node_converters/min_max.py]: > amin, amax

Fixed

**Review by ivkalgin — APPROVED** [review]

**Pull request merged**